packages/engine/Source/Scene/VoxelEllipsoidShape.js

```diff
@@ -420,9 +421,14 @@ VoxelEllipsoidShape.prototype.update = function (
     shapeMaxExtent,
     shaderUniforms.ellipsoidRadiiUv
   );
-  const axisRatio =
-    Cartesian3.minimumComponent(shapeOuterExtent) / shapeMaxExtent;
+  const { x: radiiUvX, z: radiiUvZ } = shaderUniforms.ellipsoidRadiiUv;
+  const axisRatio = radiiUvZ / radiiUvX;
   shaderUniforms.eccentricitySquared = 1.0 - axisRatio * axisRatio;
+  shaderUniforms.evoluteScale = Cartesian2.fromElements(
```

Comment:
This `evoluteScale` used to be computed in single precision on the shader, in the `nearestPointOnRadius` function in `convertUvToEllipsoid.glsl`. It is the difference of two very similar values, so it is safer to compute it in double precision on the CPU to avoid subtractive cancellation.